circle.yml

```diff
@@ -13,6 +13,7 @@ machine:
     TAG: circle-${CIRCLE_BUILD_NUM}
     RUNTIME_DISTRIBUTION: gcp-php-runtime-jessie-unstable
     BUILDER_TARGET_IMAGE: gcr.io/${GOOGLE_PROJECT_ID}/php-nginx
+    RUN_E2E_TESTS: true
```

Comment:
First I added the env on the jenkins config page, but yeah it's better to have in the script, and it's done.